app/helpers/plans_helper.rb

```diff
@@ -42,4 +42,9 @@ def visibility_tooltip(val)
       return _('Private: restricted to me and people I invite.')
     end
   end
+
+  def download_plan_page_title(plan, phase, hash)
+    # If there is more than one phase show the plan title and phase title
+    return hash[:phases].length > 1 ? "#{plan.title} - #{phase[:title]}" : plan.title
```

Comment:
@johnpinto1 

Please remove the redundant `return` here, and replace `.length > 1` with `.many?`

That should read a bit nicer :)